01_fib/main_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("0112358\n")
+	actual := strconv.Quote(buf.String())
```

Comment:
You probably don't need to call `strconv.Quote` here.

Reply:
Copy error , got rid off the unwanted strconv